### PR TITLE
Migrate Gitstars component to shadcn/tailwind

### PR DESCRIPTION
**Pull request by Baystef**
## Description
Migrated the GitStars component styling to Tailwind/Shadcn

## Related Issue
#13946

### CODE REVIEW

File: src/components/GitStars.tsx [hunk 1]
```diff
@@ -31,39 +30,18 @@ const GitStars = ({ gitHubRepo, hideStars, ...props }: GitStarsProps) => {
...
-        border="1px solid"
-        borderColor="lightBorder"
-        borderRadius="base"
-        color="text"
-        _hover={{
-          boxShadow: "0 0 1px var(--eth-colors-primary-base)",
-          path: { fill: "primary.base" },
-        }}
-      >
+      <Flex className="rounded border border-border bg-background-medium no-underline hover:fill-primary hover:shadow-table-item-box-hover">
```
Comment (pettinarip) [inline src/components/GitStars.tsx:33]: removing as it is redundant, border will default to the `border` color.
```suggestion
      <Flex className="rounded border bg-background-medium no-underline hover:fill-primary hover:shadow-table-item-box-hover">
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Good job @Baystef, left a few comments

**Review by pettinarip — APPROVED** [review]
@Baystef i've polished a bit the styles for this component. Thanks for the PR

**Comment by Baystef** [comment]
> @Baystef i've polished a bit the styles for this component. Thanks for the PR

Thanks. Is there a design you're following? @pettinarip

**Pull request merged**